js/tabs.js

```diff
@@ -0,0 +1,60 @@
+function setupTabs() {
+  var tabs = document.querySelectorAll('.js-tab');
+
+  for (var i = 0; i < tabs.length; i += 1) {
+    var tab = tabs[i];
+    var tabSelector = tab.getAttribute('href');
+    var tabContent = document.querySelector(tabSelector);
+
+    if (tabContent) {
+      tab.setAttribute('aria-selected', 'false');
+      tabContent.classList.add('u-hide');
+
+      tab.addEventListener('click', function(e) {
+        e.preventDefault();
+        toggleTab(e.currentTarget);
+      });
+    }
+  }
+
+  function toggleTab(tab) {
+    var tabSelector = tab.getAttribute('href');
+    var tabContent = document.querySelector(tabSelector);
+  
+    for (var i = 0; i < tabs.length; i += 1) {
+      if (tabs[i] !== tab) {
+        var otherTabSelector = tabs[i].getAttribute('href');
+        var otherTabContent = document.querySelector(otherTabSelector);
+        
+        tabs[i].setAttribute('aria-selected', 'false');
+        tabs[i].classList.remove('active');
+        otherTabContent.classList.add('u-hide');
+      }
+    }
+  
+    tab.setAttribute('aria-selected', 'true');
+    tab.classList.toggle('active');
+    tabContent.classList.toggle('u-hide');
+  
+    if (tab.classList.contains('active')) {
+      var breakpoint = 620;
+      var yOffset = 200;
+      var rect = tabContent.getBoundingClientRect();
+      var inView = ((window.innerHeight - yOffset) > rect.top);
+      var scrollTarget = window.scrollY + yOffset;
+  
+      if (window.innerWidth <= breakpoint) {
+        scrollTarget = (pageYOffset + rect.top) - 10;
```

Comment:
This is still scrolling more than it needs to:
- With the buttons on one line at the bottom of the screen, clicking one of them scrolls to reveal step 1 and part of 2 — then clicking another button scrolls even further, when it needn’t.
- When the buttons wrap to two lines, clicking any of them scrolls, as before, so far that the buttons disappear off-screen altogether.

But it’s much improved, so I won’t hold this up further.